apps/cyberstorm-storybook/.storybook/preview.js

```diff
@@ -1,5 +1,6 @@
 import { LinkLibrary } from "../LinkLibrary";
 import { LinkingProvider } from "@thunderstore/cyberstorm";
+import "@thunderstore/cyberstorm-styles/dist/thunderstore-cyberstorm-styles.esm";
```

Comment:
Would this work with just `import "@thunderstore/cyberstorm-styles";`? The user of the package shouldn't be this aware of the internal structure?

Reply:
fixed 👍